site/source/support/faq/faq-bitcoin.rst

```diff
@@ -32,49 +32,49 @@ Many wallets also do not yet support pruned nodes, which is one big reason that
 
 Is it insecure to run a pruned node?
 ------------------------------------
-As a user, pruned nodes and archival nodes provide you the same security.  In a larger sense, if 100% of people ran pruned nodes, the security of the network could be in dire circumstances and be put at risk if no nodes kept history, as then no one could bootstrap new nodes.  The reality however, is that most Embassy owners are new node operators, so there is no net systemic risk introduced if users decide to run a pruned node.
+As a user, pruned nodes and archival nodes provide you the same security.  In a larger sense, if 100% of people ran pruned nodes, the security of the network could be in dire circumstances and be put at risk if no nodes kept history, as then no one could bootstrap new nodes.  The reality however, is that most StartOS users are new node operators, so there is no net systemic risk introduced if users decide to run a pruned node.
 
 The Bitcoin Service is different from the GUI version I am used to using (bitcoin-qt).  How do I use this like I used to?
 -------------------------------------------------------------------------------------------------------------------------
-At present, the Embassy does not offer its own node visualization tool. You can view certain properties about your node inside the "Properties" section, but not in an animated GUI. If you want to use bitcoin-cli, you may do so by adding an SSH key onto your Embassy and :ref:`exec-ing into the bitcoind docker container<service-container>`. Otherwise, the main way to actually *use* your node is through a wallet. The QT GUI is not usable because it cannot be served through the browser (which is necessary here), and last we checked, the QT desktop client was incapable of connecting to a remote node over Tor.
+At present, StartOS does not offer its own node visualization tool. You can view certain properties about your node inside the "Properties" section, but not in an animated GUI. If you want to use bitcoin-cli, you may do so by adding an SSH key to your server and :ref:`exec-ing into the bitcoind docker container<service-container>`. Otherwise, the main way to actually *use* your node is through a wallet. The QT GUI is not usable because it cannot be served through the browser (which is necessary here), and last we checked, the QT desktop client was incapable of connecting to a remote node over Tor.
```

Comment:
Specter is basically a bitcoin core gui - too much of a stretch?

Reply:
Fine as is